test/unit/test-static-route.js

```diff
@@ -1,14 +1,67 @@
 var assert = require('assert');
 var request = require('supertest');
-var app = require('../../application.js');
+var proxyquire = require('proxyquire');
 
-exports['test returned months data'] = function (done) {
-  request(app)
-  .get('/assets').expect(200).end(function (err, response) {
-    var body = response.body;
-    assert(Array.isArray(body), 'response body should be an array');
-    assert.equal(body[0].key, 0, 'first key is 0');
-    assert.equal(body[0].value, 'Office Chairs', 'first value is Office Chairs');
-    return done();
-  });
+var corsMock = function () {
+  return function (req, res, next) {
+
+    return next();
+  };
+};
+var mbaasApiStub = {
+  mbaasExpress: function () {
+    return {
+      sys: function () {
+        return function (req, res, next) { return next(); };
+      },
+      mbaas: function (req, res, next) {
+        return next();
+      },
+      fhmiddleware: function () {
+        return function (req, res, next) {
+          return next();
+        };
+      },
+      cloud: function () {
```

Comment:
`cloud` is not in https://github.com/feedhenry-templates/appforms-datasources-mongodb/blob/master/application.js ?